### PR TITLE
content: ockam c coding standard md file

### PR DESCRIPTION
**Pull request by mwiszniewski**
Version 1.0

### CODE REVIEW

**Comment by mattgreg** [comment]
we require that all commit messages are in lowercase.

also can you make this commit message `content:` instead of `docs:`

File: src/content/learn/guides/Contributing/Ockam_C_Coding_Standard.md [hunk 1]
```diff
@@ -0,0 +1,878 @@
+# C ​ Coding Standard
+
+**Adapted from** ​ **[http://www.possibility.com/Cpp/CppCodingStandard.html](http://www.possibility.com/Cpp/CppCodingStandard.html)** ​ **and NetBSD's
```
Comment (mattgreg) [inline src/content/learn/guides/Contributing/Ockam_C_Coding_Standard.md:3]: In our other guides we use a structure where at the bottom of the document we have a section called 
`## References`
Let's create a references H2 section at the bottom and put this text there to be similar to our other guide structures.

**Comment by mattgreg** [comment]
Can you also squash this PR to one commit?

File: src/content/learn/guides/Contributing/Ockam_C_Coding_Standard.md (hunk 1 above)
Comment (mwiszniewski) [reply src/content/learn/guides/Contributing/Ockam_C_Coding_Standard.md:3]: Solved in new commit.

**Comment by mwiszniewski** [comment]
> we require that all commit messages are in lowercase.
> 
> also can you make this commit message `content:` instead of `docs:`



> we require that all commit messages are in lowercase.
> 
> also can you make this commit message `content:` instead of `docs:`

Solved in new commit.

**Comment by mwiszniewski** [comment]
> Can you also squash this PR to one commit?

I couldn't make sqhasu so i pushed my change again and created new pull request. This one can be closed.

**Pull request closed without merging**